types/defines/trace.d.ts

```diff
@@ -149,7 +149,7 @@ interface Exception {
 interface Log {
   readonly type: "log";
   readonly level: "debug" | "error" | "info" | "log" | "warn";
-  readonly message: string;
+  readonly message: object;
```

Comment:
Are we certain that parsing the `message` will always produce an object? As far as I see in the `parseJson(...)` login in `trace-stream` we're not verifying that it is indeed an object that is the result of the parse?

Reply:
Good question – in the LTW (see api/trace.c++ getTraceLogMessage()) we cast the return value of js.parseJson() to v8::Object, does that enforces that we're returning an object?